vitest.base.unit.config.ts

```diff
@@ -42,5 +42,6 @@ export default defineConfig({
       ],
     },
     diff: process.env.TEST_COMPACT_DIFF ? path.join(import.meta.dirname, "./scripts/vitest/vitest.diff.ts") : undefined,
+    onConsoleLog: () => process.env.TEST_QUIET_CONSOLE !== "true",
```

Comment:
This will be expensive check, as for every log message this function will be called. Not fully against it, but we should document somewhere that it's a expensive.

Reply:
It should be neglectable compared to the cost of logging a line to the console. I haven't notices any slowdown.

Reply:
shouldn't be checking against `"true"` here